.travis.yml

```diff
@@ -10,17 +10,19 @@
 # SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
 #
 
+dist: xenial
 language: java
 sudo: required
 
+services:
+    - mysql
```

Comment:
I think I read that only "services" is needed if "addons" isnt being used. Not sure about "sources"... but maybe "addons" can be removed completely if we are using "services" for mysql now.

Reply:
if it is not listed in services then the db server won't be started by default (from xenial), that's how I understand the doc.

I don't think libmysql-java is there by default, hence we likely need addons/apt/packages at least

Reply:
Ok, then maybe "mysql-server" in "services" is just redundant

Reply:
I was reading this doc to see what needs to be changed for xenial: https://blog.travis-ci.com/2019-04-15-xenial-default-build-environment